src/components/CmsInspectSheet.vue

```diff
@@ -0,0 +1,224 @@
+          <div v-else>Nothing was delivered to this zone.</div>
+        </div>
+
+        <div class="cms-inspect-sheet__section">
+          <h4>Context</h4>
+          <template v-if="renderContextEntries.length > 0">
+            <div>This zone has access to the following extra fields in the CMS template:</div>
+            <div
+              v-for="(entry, index) in renderContextEntries"
+              :key="index"
+              class="cms-inspect-sheet__row"
+            >
+              <div class="cms-inspect-sheet__row__label">{{ entry[0] }}</div>
+              <div class="cms-inspect-sheet__row__value">{{ entry[1] }}</div>
+            </div>
+          </template>
+          <div v-else>There are no additional fields for this zone.</div>
+        </div>
+
+        <div class="cms-inspect-sheet__section">
+          <h4>Sitevars</h4>
+          <div>These variables are shared across all zones:</div>
+          <div v-for="(entry, index) in siteVarEntries" :key="index" class="cms-inspect-sheet__row">
+            <div class="cms-inspect-sheet__row__label">{{ entry[0] }}</div>
+            <div class="cms-inspect-sheet__row__value">{{ entry[1] }}</div>
+          </div>
+        </div>
+      </div>
+
+      <div class="cms-inspect-sheet__footer">
+        <a class="cms-inspect-sheet__button" target="_blank" :href="zoneAdminLink">
+          View in admin
+        </a>
+        <button
+          class="cms-inspect-sheet__button cms-inspect-sheet__button--text"
+          @click="$emit('input', false)"
+        >
+          Close
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import { Content } from '../api';
+import { pluginOptions } from '../plugins/cms';
+
+@Component
+export default class CmsInspectSheet extends Vue {
+  @Prop({ type: Boolean, default: false }) value!: boolean;
+  @Prop({ type: String, required: true }) zoneId!: string;
+  @Prop({ type: String }) zoneStatus?: string;
+  @Prop({ type: String }) zoneType?: string;
+  @Prop({ type: String }) zoneHeader?: string;
+  @Prop({ type: String }) zoneFooter?: string;
+  @Prop(Object) renderContext?: object;
+  @Prop({ type: Array, default: () => [] }) contents!: Content[];
+
+  siteVars = {};
+
+  created(): void {
+    this.siteVars = pluginOptions.getSiteVars();
+  }
+
+  get zoneAdminLink(): string {
+    return `${pluginOptions.baseUrl}/admin/zone/edit/?id=64`;
+  }
+
+  get siteVarEntries() {
+    return Object.entries(this.siteVars).filter((entry) => {
+      return entry[1] !== undefined && entry[1] !== null && entry[1] !== '';
+    });
+  }
+
+  get renderContextEntries() {
+    return Object.entries(this.renderContext || {}).filter((entry) => {
+      return entry[1] !== undefined && entry[1] !== null;
+    });
+  }
+
+  getDeliveryAdminLink(deliveryId: number): string {
+    return `${pluginOptions.baseUrl}/admin/delivery/?flt0_0=${deliveryId}`;
```

Comment:
Nice!!! :heart:

Reply:
Might also want to include the link to the content edit page if that's easy

Reply:
I don't think we actually get the content ID in the server response but I do agree that'd be a nice inclusion.